src/Link.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Text from './Text'
+
+const anchorText = Text.withComponent('a')
+
+const Link = styled(anchorText)`
+  cursor: ${props => props.cursor};
+  text-decoration: ${props => props.textDecoration};
+  display: ${props => props.display};
+`
+
+Link.displayName = 'Link'
+
+Link.defaultProps = {
+  cursor: 'pointer',
+  textDecoration: 'none',
+  display: 'inline-block'
+}
+
+Link.propTypes = {
+  cursor: PropTypes.string,
+  textDecoration: PropTypes.string,
+  display: PropTypes.string
```

Comment:
I wouldn't add these as props, if someone needs to extend the `Link` component in their app, they can handle that on their own. Are there parts of the design mocks where you see a need for things like this?

Reply:
Took a step back and re-thought about the designs using `Link`. Think it'd be safer to have the host app define these specific styles when using `Link`. No need to maintain the props here to support specific styles.

Retired these props' code to simple further. Please review again at your convenience. 👍